doc/orchestrator.rst

```diff
@@ -166,325 +166,388 @@ A full example of configuring KeyDB to run in SmartSim is shown below
 
   # run smartsim workload
 
-Multiple Orchestrators
-======================
-
+Multi-Orchestrator Example
+==========================
 SmartSim offers functionality to automate the deployment of multiple
 databases on an HPC cluster, supporting workloads that require multiple
 ``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
-simulation with high inference performance demands, along with an analysis and
-visualization workflow connected to the simulation. In such cases, SmartSim
-allows you to augment the number of clustered orchestrators or provision multiple
-co-located orchestrators, ensuring that the workload can scale to meet its resource
-requirements. Below is a general example demonstrating the process of launching
-multiple databases with unique identifiers and distributing data between them.
-
-.. note::
+simulation with high inference performance demands (necessitating a co-located deployment),
+along with an analysis and
+visualization workflow connected to the simulation (requiring a standard orchestrator).
+
+Below is a simple example demonstrating the process of:
+
+1. Launch two standard Orchestrators within the experiment with unique identifiers
+2. Launch one colocated Orchestrator within the experiment with a unique identifier
+3. Connect SmartRedis clients within the driver script to send tensors to std Orchestrators
+4. Connect SmartRedis clients within the application to retrieve tensors from std databases to store in colo databases
```

Comment:
Make these gerunds: Launching, Connecting

Reply:
yes sirrr!

Reply:
delete the 1 orchestrator